### DISCUSSION TITLE
rabbitmq-diagnostics -q status error with rabbitmq:3.11.0

### DISCUSSION THREAD

**Discussion by Broderick890** [rabbitmq/rabbitmq-server#6045] [Other]
node info:rabbitmq1, rabbitmq2, rabbitmq3, rabbitmq4 , rabbitmq5
for some reason , rabbitmq2 node machine  reboot, when rabbitmq2  back to  the cluster ,the cluster throw error below

Error:
{:badarg, [{:erlang, :++, [{:badrpc, {:EXIT, {:badarg, [{:ets, :match_object, [:rabbit_listener_ets, :_], [error_info: %{cause: :id, module: :erl_stdlib_errors}]}, {:ets, :tab2list, 1, [file: 'ets.erl', line: 771]}]}}}, [{:listener, :rabbit@rabbitmq1, :http, 'rabbitmq1', {0, 0, 0, 0, 0, 0, 0, 0}, 15672, [cowboy_opts: [sendfile: false], port: 15672]}, {:listener, :rabbit@rabbitmq1, :"http/prometheus", 'rabbitmq1', {0, 0, 0, 0, 0, 0, 0, 0}, 15692, [cowboy_opts: [sendfile: false], port: 15692, protocol: :"http/prometheus"]}, {:listener, :rabbit@rabbitmq1, :clustering, 'rabbitmq1', {0, 0, 0, 0, 0, 0, 0, 0}, 25672, []}, {:listener, :rabbit@rabbitmq1, :amqp, 'rabbitmq1', {0, 0, 0, 0, 0, 0, 0, 0}, 5672, [backlog: 128, nodelay: true, linger: {true, 0}, exit_on_close: false]}]], [error_info: %{module: :erl_erts_errors}]}, {:lists, :append, 1, [file: 'lists.erl', line: 155]}, {:lists, :append, 1, [file: 'lists.erl', line: 155]}, {:rabbit, :listeners, 0, [file: 'rabbit.erl', line: 765]}, {:rabbit, :status, 0, [file: 'rabbit.erl', line: 706]}]}
root@rabbitmq1:/var/log/rabbitmq# rabbitmq-diagnostics -q alarms
Node rabbit@rabbitmq1 reported no alarms, local or clusterwide

Cluster work fine before rabbitmq2   back to the cluster

**Comment by michaelklishin** [comment]
I cannot reproduce. We need more information about what node the command is executed against.

The exception very likely means that an in-memory table is not yet initialized, which for a booting node can be the case. In fact, it comes from a standard Erlang module and not RabbitMQ.
